packages/cli/src/ui/hooks/slashCommandProcessor.ts

```diff
@@ -153,6 +154,41 @@ export const useSlashCommandProcessor = (
     [addItem],
   );
 
+  const reloadMcpAction = useCallback(async () => {
```

Comment:
nit: should we update all `reload` usage to `refresh` to be consistent with command itself.
```suggestion
  const refreshMcpAction = useCallback(async () => {
```